config/identity/config.yaml

```diff
@@ -162,3 +166,18 @@ ci-issuer-metadata:
         run-invocation-uri: "{{ .url }}/{{ .project_path }}/-/jobs/{{ .job_id }}"
         source-repository-visibility-at-signing: "repository_visibility"
     subject-alternative-name-template: "https://{{ .ci_config_ref_uri }}"
+  *codefresh-type:
+    default-template-values:
+        url: "https://g.codefresh.io"
+    extension-templates:
+        build-signer-uri: "{{if .platform_url}}{{.platform_ur}}{{ else }}{{.url}}{{end}}/build/{{ .workflow_id }}"
```

Comment:
Can we copy in the comments from the principal files, like https://github.com/sigstore/fulcio/blob/main/pkg/identity/codefresh/principal.go#L153-L154 and the ones in buildkite as well?